pkg/operator/ceph/cluster/controller.go

```diff
@@ -309,10 +298,6 @@ func (c *ClusterController) onAdd(obj interface{}) {
 	osdChecker := osd.NewMonitor(c.context, cluster.Namespace)
 	go osdChecker.Start(cluster.stopCh)
 
-	// Start the ceph status checker
-	cephChecker := newCephStatusChecker(c.context, cluster.Namespace, clusterObj.Name)
-	go cephChecker.checkCephStatus(cluster.stopCh)
```

Comment:
Where did the ceph checker go?

Reply:
I'm also not really sure why this is removed. @shaas was this an accidental deletion instead of a move, or is this not needed any more?

Reply:
Upps... That was not intended.